src/compiler.jl

```diff
@@ -1720,6 +1939,8 @@ function GPUCompiler.codegen(output::Symbol, job::CompilerJob{<:EnzymeTarget};
         augmented_primalf = nothing
     end
 
+    # fix_inlining!(mod)
```

Comment:
So we can remove `fix_inling` from this PR?

Reply:
it was to attempt to fix mpi misbehaving, but I don't think it ended up being that issue. Regardless fine to split with MPI